src/helpers.js

```diff
@@ -4,10 +4,14 @@ const PI = Math.PI;
 const HALF_PI = PI / 2;
 
 export function scaleValue(scale, value, fallback) {
-  value = typeof value === 'string' ? scale.parse(value) : value;
+  value = valueAsNumber(scale, value);
   return isFinite(value) ? scale.getPixelForValue(value) : fallback;
 }
 
+export function valueAsNumber(scale, value) {
+  return typeof value === 'string' ? scale.parse(value) : value instanceof Date ? value.getTime() : value;
```

Comment:
Why not  use the `scale.parse` for date?

Reply:
I didn't know it... I'll change it.

Reply:
> 
> 
> Why not use the `scale.parse` for date?

I had a look to `parse` method. Why don't use it for all types (also number)? 
It sounds working

Reply:
because of the fallback logic. but maybe that can be done differently.

Reply:
> 
> 
> because of the fallback logic. but maybe that can be done differently.

Let me try if I can.